src/mad_spark_alt/evolution/fitness.py

```
@@ -12,16 +12,48 @@
 from mad_spark_alt.core.interfaces import GeneratedIdea
 from mad_spark_alt.core.unified_evaluator import HypothesisEvaluation, UnifiedEvaluator
 from mad_spark_alt.evolution.interfaces import (
+    DiversityMethod,
     EvolutionConfig,
     IndividualFitness,
+    PopulationSnapshot,
```

Ruff (0.12.2)

`mad_spark_alt.evolution.interfaces.PopulationSnapshot` imported but unused

Remove unused import: `mad_spark_alt.evolution.interfaces.PopulationSnapshot`

(F401)